paasta_tools/cli/cmds/spark_run.py

```diff
@@ -537,14 +548,29 @@ def configure_and_run_docker_container(
     system_paasta_config: SystemPaastaConfig,
     spark_conf: Mapping[str, str],
     aws_creds: Tuple[Optional[str], Optional[str], Optional[str]],
+    cluster_manager: str,
 ) -> int:
 
     # driver specific volumes
-    volumes = (
-        spark_conf.get("spark.mesos.executor.docker.volumes", "").split(",")
-        if spark_conf.get("spark.mesos.executor.docker.volumes", "") != ""
-        else []
-    )
+    if cluster_manager == CLUSTER_MANAGER_MESOS:
```

Comment:
Can we add an info log-line somewhere, to display which cluster-manager is being used?